### PR TITLE
[Tizen] Fix wrong condition checking

### PR DESCRIPTION
**Pull request by piotrowskim**
In function ContentInstance::HandleSyncMessage there is wrong
condition check, which is always true.

### CODE REVIEW

File: src/content/content_instance.cc [hunk 1]
```diff
@@ -145,9 +145,8 @@ void ContentInstance::PostAsyncSuccessReply(const picojson::value& msg,
 
 void ContentInstance::HandleSyncMessage(const char* message) {
   picojson::value v;
-  picojson::value::object o;
-
   std::string err;
+
```
Comment (rakuco) [inline src/content/content_instance.cc]: I don't think this is necessary, especially because those variables are all used in the following line anyway. As weird as it may look if you're not used to Chromium's coding style, it is geared towards minimizing [vertical whitespace](https://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Vertical_Whitespace).

File: src/content/content_instance.cc [hunk 2]
```diff
@@ -145,9 +145,8 @@ void ContentInstance::PostAsyncSuccessReply(const picojson::value& msg,
 
 void ContentInstance::HandleSyncMessage(const char* message) {
   picojson::value v;
-  picojson::value::object o;
-
```
Comment (rakuco) [inline src/content/content_instance.cc]: This looks unrelated to what you say your commit does.

File: src/content/content_instance.cc [hunk 3]
```diff
@@ -756,20 +755,21 @@ void ContentItem::print(void) {
 #endif
 
 void ContentInstance::HandleScanFileRequest(const picojson::value& msg) {
-  if (msg.contains(STR_CONTENT_URI)) {
```
Comment (rakuco) [inline src/content/content_instance.cc]: I don't see any functional change here. Removing indentation levels does make sense, and I'd certainly support writing it this way if you were adding the code, but changing it just for aesthetic purposes isn't very useful and makes looking at the commit history more complicated.

File: src/content/content_instance.cc (hunk 1 above)
Comment (piotrowskim) [inline src/content/content_instance.cc]: Ok, will remove new line.

File: src/content/content_instance.cc (hunk 2 above)
Comment (piotrowskim) [inline src/content/content_instance.cc]: What is better: new commit or amending commit message?

File: src/content/content_instance.cc (hunk 3 above)
Comment (piotrowskim) [inline src/content/content_instance.cc]: Since whole body was in one "if", I decided to change it. But will revert it.

File: src/content/content_instance.cc (hunk 2 above)
Comment (rakuco) [inline src/content/content_instance.cc]: New commit. Even better, new pull request, as it's got nothing to do with the rest :)

Comment (piotrowskim) [inline src/content/content_instance.cc]: Ok, new commit even better:)

**Comment by rakuco** [comment]
lgtm, thanks! If you're in the mood, please consider adding a test case to crosswalk-test-suite.

**Comment by rakuco** [comment]
One suggestion which only occurred to me after I merged your commit: since this is tizen-extensions-crosswalk, saying "[Tizen]" in the first line doesn't add much information. It's more helpful if you use the extension instead, like "[Content] Fix wrong condition check".

**Comment by piotrowskim** [comment]
Ok, I will adjust next commit message to your suggestion.
